### ISSUE TITLE
Is this still being maintained?

### ISSUE THREAD

**Issue by jankonas** [neomerx/cors-psr7#53]
Hi @neomerx, do you have any plans with this repository? I see you are no longer active on Github and there are a year old issue and PR.

Just asking if there is any chance of getting some tiny PR merged If I create one. :-)

**Comment by mbolli** [comment]
@neomerx yes a tagged release with psr/http-message 2.x support would be awesome!

**Comment by filisko** [comment]
Hi @mbolli @jankonas @neomerx, we're planning to fork this repo in `middlewares/*`. Keep up to date 💪 

- https://github.com/middlewares/cors/issues/9#issuecomment-3563341257
